packages/conf-gtksourceview3/conf-gtksourceview3.0/opam

```diff
@@ -14,7 +14,7 @@ depexts: [
   ["gtksourceview3-devel"] {os-distribution = "fedora"}
   ["gtksourceview3"] {os = "freebsd"}
   ["gtksourceview3"] {os = "openbsd"}
-  ["gtksourceview3-devel"] {os-family = "suse"}
+  ["gtksourceview-devel"] {os-family = "suse"}
```

Comment:
Is the gtksourceview3 library always been called gtksourceview on suse, or it is a "recent" change? In the second case can you please move this change to `conf-gtksourceview3.1`?

Reply:
No clue, their package DB seems not to provide this info. @clarus do you know?

Reply:
As the package refers to the `gtksourceview3.0` library (see pkgconfig call), wouldn't `conf-gtksourceview3.0+1` or such be more appropriate? Otherwise, what'd be the name for depending on pkgconfig's `gtksourceview-3.1`?

Reply:
Pro